src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1394,86 +1470,73 @@ void invalid() {
-                final Resource eventEdge = markAsArgument(model, event, SeedlingOntology.Conflict_Attack_Target,
-                        entity, system, 1.0, getAssertionUri());
-                markJustification(eventEdge, justification);
-
-
-                final Resource justification1 = makeTextJustification(model, "source1", 0, 4, system, 1d);
-                final Resource compound1 = markCompoundJustification(model,
-                        ImmutableSet.of(entity, relation, event),
-                        ImmutableSet.of(justification1),
-                        system,
-                        1d);
-
-                markJustification(entity, compound1);
-                markJustification(relation, compound1);
-                markJustification(event, compound1);
-
-                assertAndDump(model, "NIST.invalid: CompoundJustification must be used only for " +
-                                "justifications of argument assertions",
-                        nistSeedlingValidator, false);
-
+                testValid("NIST.invalid: Cluster has IRI");
             }
+
             @Test
             void valid() {
+                testValid("NIST.valid: Cluster has IRI");
+            }
+        }
+    }
 
-                // test relation argument
-                final Resource relation = makeRelation(model, getUri("relationX"), system);
-                addType(relation, SeedlingOntology.GeneralAffiliation_APORA);
-                makeClusterWithPrototype(model, getClusterUri(), relation, system);
-                final Resource relationEdge = markAsArgument(model, relation,
-                        SeedlingOntology.GeneralAffiliation_APORA_Affiliate, entity, system, 1d);
-                final Resource justification1 = makeTextJustification(model, "source1", 0, 4, system, 1d);
-                final Resource compound = markCompoundJustification(model,
-                        ImmutableSet.of(relationEdge),
-                        ImmutableSet.of(justification1),
-                        system,
-                        1d);
-
-                markJustification(relationEdge, compound);
+    /**
+     * Set of tests to show that NIST Hypothesis restrictions pass and fail appropriately
+     */
+    @Nested
+    class NISTHypothesisExamples extends BaseExample {
+        Resource entity;
+        Resource entityCluster;
+        NISTHypothesisExamples() { validator = nistSeedlingHypothesisValidator; }
 
-                // test event argument
-                final Resource eventEdge = markAsArgument(model, event, SeedlingOntology.Conflict_Attack_Target, entity, system, 1.0);
+        @BeforeEach
+        void setup() {
+            super.setup();
+            entity = makeEntity(model, getEntityUri(), system);
+            addType(entity, SeedlingOntology.Person);
+            entityCluster = makeClusterWithPrototype(model, getClusterUri(), entity, system);
+        }
```

Comment:
I guess `NISTHypothesisExamples` would also have to @override `testValid` and `testInvalid` if you took my suggestion from before.